GVFS/GVFS.Common/Maintenance/LooseObjectsStep.cs

```diff
@@ -65,15 +103,18 @@ protected override void PerformMaintenance()
                         }
                     }
 
-                    int beforeCount = this.CountLooseObjects();
+                    int beforeLooseObjectsCount = this.CountLooseObjects();
                     GitProcess.Result result = this.RunGitCommand((process) => process.PrunePacked(this.Context.Enlistment.GitObjectsRoot));
-                    int afterCount = this.CountLooseObjects();
+                    int afterLooseObjectsCount = this.CountLooseObjects();
+
+                    this.CreateLooseObjectsPackFile();
 
                     EventMetadata metadata = new EventMetadata();
                     metadata.Add("GitObjectsRoot", this.Context.Enlistment.GitObjectsRoot);
-                    metadata.Add("StartingCount", beforeCount);
-                    metadata.Add("EndingCount", afterCount);
-                    metadata.Add("RemovedCount", beforeCount - afterCount);
+                    metadata.Add("StartingCount", beforeLooseObjectsCount);
+                    metadata.Add("EndingCount", afterLooseObjectsCount);
+                    metadata.Add("RemovedCount", beforeLooseObjectsCount - afterLooseObjectsCount);
+                    metadata.Add(nameof(this.LooseObjectsPutIntoPackFile), this.LooseObjectsPutIntoPackFile);

```

Comment:
nit: let's not make this nameof here, so that refactoring doesn't change what we look for in the data exhaust streams.